lib/ansible/modules/windows/win_chocolatey.py

```diff
@@ -51,6 +51,16 @@
   source:
     description:
       - Specify source rather than using default chocolatey repository.
+  user:
+    description:
+      - Username used to authenticate against a source.
+    default: 'no'
```

Comment:
Take the default key out as nothing is actually set, `no` means `False` which in this case is incorrect.